tests/Parser/Representation/RepresentationParserTest.php

```diff
@@ -134,6 +135,33 @@ public function testRepresentationMethodsThatWillFailParsing($class, $method, $e
         }
     }
 
+    public function testRepresentationThatHasVersioningAcrossMultipleAnnotations()
+    {
+        $class = '\Mill\Tests\Fixtures\Representations\RepresentationWithVersioningAcrossMultipleAnnotations';
+        $parser = new RepresentationParser($class);
+        $annotations = $parser->getAnnotations('create');
+
+        $this->assertSame([
+            'connections',
+            'connections.things',
+            'connections.things.name',
+            'connections.things.uri',
+            'unrelated'
+        ], array_keys($annotations));
+
+        $this->assertSame('>=3.3', $annotations['connections']->toArray()['version']);
+        $this->assertSame('>=3.3', $annotations['connections.things']->toArray()['version']);
+        $this->assertSame('>=3.3', $annotations['connections.things.name']->toArray()['version']);
+        $this->assertSame('3.4', $annotations['connections.things.uri']->toArray()['version']);
+        $this->assertEmpty($annotations['unrelated']->toArray()['version']);
+
+        $this->assertEmpty($annotations['connections']->toArray()['capability']);
+        $this->assertSame('NONE', $annotations['connections.things']->toArray()['capability']);
+        $this->assertSame('NONE', $annotations['connections.things.name']->toArray()['capability']);
+        $this->assertSame('NONE', $annotations['connections.things.uri']->toArray()['capability']);
+        $this->assertEmpty($annotations['unrelated']->toArray()['version']);
+    }
+
```

Comment:
Could you add a test or another assertion 
 - when a capability is set on a parent annotation? 
 - when a child has a different capability than the parent?